cdap-runtime-ext-dataproc/pom.xml

```diff
@@ -98,7 +95,6 @@
     <dependency>
       <groupId>com.google.api-client</groupId>
```

Comment:
This dependency was added explicitly because `google-api-services-compute` was using an older version but if we can remove it since now `google-api-services-compute` also uses `2.2.0`.

Reply:
Addressing in the follow-up https://github.com/cdapio/cdap/pull/15323